hangman.rb

```diff
@@ -0,0 +1,31 @@
+random_words = ["camera", "photo", "picture", "landscape", "light", "shadow", "lens", "portrait"]
+#the number is one more
+random_num = rand(0..7).to_i
+chosen_word = random_words[random_num]
+letter_array = chosen_word.chars
+blank_array = [ ]
+guesses = 7
+state = true
+for i in 0..letter_array.length-1
+  blank_array.push(" __ ")
+end
```

Comment:
Going to teach you a trick or the "Ruby Way". So we actually do loops differently in Ruby. We use `for` `each`. So let me show you how it works.

```rb
letter_array.each do |letter|
  puts letter
end

# This would display one letter at a time.
```

```rb
[1,2,3,4].each do |i|
  puts i * 2
end

# This would display the following
#
# 2
# 4
# 6
# 8
```